app/helpers/view_hook_helper.rb

```diff
@@ -0,0 +1,13 @@
+module ViewHookHelper
+
+  # Look up the current path and render the view hooks for a specific placement
+  def view_hook(placement, args = {})
```

Comment:
Yeah, I'm definitely not sold on all the names I've come up with so far. This PR (and the other two) are really more an exploration of the concept. There's obviously cleanup and some refactoring left to do, but I didn't want to spend too much time on that since some of these spikes will just be thrown away.